### PR TITLE
website: add page on blsync

### PR DESCRIPTION
**Pull request by s1na**

### CODE REVIEW

File: docs/fundamentals/blsync.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Beacon light client
+description: Running geth with integrated beacon light client
+---
+
+`blsync` is a beacon light client shipped by Geth, designed to synchronize with the beacon chain without the need for a [consensus client](/docs/getting-started/consensus-clients). Integrated within Geth, blsync eliminates the necessity of running a separate consensus client, making it ideal for most use-cases that do not require full validation capabilities. It comes with very low resource requirements and can sync the beacon chain within seconds. `blsync` can be run in two modes: integrated or standalone. In standalone mode it is possible to use it for driving other execution clients.
```
Comment (MariusVanDerWijden) [inline docs/fundamentals/blsync.md:6]:
```suggestion
`blsync` is a beacon light client shipped by Geth, designed to synchronize with the beacon chain without the need for a [consensus client](/docs/getting-started/consensus-clients). Integrated within Geth, blsync eliminates the necessity of running a separate consensus client, making it ideal for use-cases that do not require full validation capabilities. It comes with very low resource requirements and can sync the beacon chain within seconds. `blsync` can be run in two modes: integrated or standalone. In standalone mode it is possible to use it for driving other execution clients.
```

File: docs/fundamentals/blsync.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+#### Running
+
+Blsync takes the same flags as above to configure the HTTP endpoint as well as checkpoint. It additionally needs flags to connect to the execution client. Specifically `--blsync.engine.api` to configure the Engine API url and `--blsync.jwtsecret` for the JWT authentication token.
+
+Again to sync the Sepolia network in this mode, first run Geth:
+
+```terminal
+❯ ./build/bin/geth --sepolia --datadir light-sepolia-dir
```
Comment (MariusVanDerWijden) [inline docs/fundamentals/blsync.md:59]:
```suggestion
./build/bin/geth --sepolia --datadir light-sepolia-dir
```

**Review by MariusVanDerWijden — APPROVED** [review]

**Comment by s1na** [comment]
I'll wait for @zsfelfoldi to take a look before merging

File: docs/fundamentals/blsync.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+./blsync --sepolia --beacon.api https://sepolia.lightclient.xyz --beacon.checkpoint 0x0014732c89a02315d2ada0ed2f63b32ecb8d08751c01bea39011b31ad9ecee36 --blsync.engine.api http://localhost:8551 --blsync.jwtsecret light-sepolia-dir/geth/jwtsecret
+
+INFO [06-23|15:06:33.388] Loaded JWT secret file                   path=light-sepolia-dir/geth/jwtsecret crc32=0x5a92678
+INFO [06-23|15:06:34.130] Successful NewPayload                    number=6,169,314 hash=d4204e..772e65 status=SYNCING
+INFO [06-23|15:06:34.130] Successful ForkchoiceUpdated             head=d4204e..772e65 status=SYNCING
+```
+
+## Summary
+
+This guide should provide the steps of using blsync in both integrated and standalone modes with Geth as an alternative to running a full consensus client. This is suitable for non-production use-cases and offers lower resource requirements and higher speed. The steps to run blsync include choosing an endpoint, obtaining a checkpoint, and starting the client with correct flags.
```
Comment (fjl) [inline docs/fundamentals/blsync.md:74]: This whole paragraph is useless. ChatGPT just loves to repeat itself I guess.

Comment (s1na) [reply docs/fundamentals/blsync.md:74]: Haha I just tried to follow the structure of other pages, so explicitly asked chat to write a summary :)

Comment (fjl) [reply docs/fundamentals/blsync.md]: I have removed it. The page isn't very long and IMHO doesn't need a summary. Also sounds too much like AI.

**Pull request merged**